algo/src/main/java/com/hartwig/serve/extraction/ExtractionResultWriter.java

```diff
@@ -43,6 +48,11 @@ public ExtractionResultWriter(@NotNull final String outputDir, @NotNull final Re
     }
 
     public void write(@NotNull ExtractionResult result) throws IOException {
+        writeTsv(result);
```

Comment:
suggestion, we could also do this with a feature toggle approach where you turn on JSON support? What do you think @cbruel @kduyvesteyn ?

Reply:
The advantage being you wouldn't need to cart around TSV if you are using JSON and vice-versa

Reply:
The TSV should be removed, shouldn't be a need to have them anymore.

We do need to keep the hotspot VCF and the interpretation TSV btw, but do think they could be separate outputs (they are not needed by the typical SERVE users